packages/GA4Client/src/event-handler.js

```diff
@@ -89,12 +91,17 @@ EventHandler.prototype.logPageView = function (event) {
         } else if (event.CustomFlags.hasOwnProperty(LEGACY_GA_LOCATION)) {
             pageLocation = event.CustomFlags[LEGACY_GA_LOCATION];
         }
+
+        if (event.CustomFlags.hasOwnProperty(REFERRER)) {
```

Comment:
Does `REFERRER` not have a legacy custom flag? I assume the answer is no, but I wanted to raise the question.

Reply:
Mo mentions this in the PR summary: 
`This flag has no legacy backwards compatibility as it wasn't available for GA Universal as you can see in the screenshot below taken from this GA support doc`

Reply:
@rmi22186 so it turned out there was a legacy backwords compatibility for referrer and google docs contradicting each other after confirming with @alexs-mparticle in hacker hour today and I did add it